includes/devices_manager.inc

```diff
@@ -146,6 +146,12 @@ switch ($sub_type) {
     case "rebuild_selected_phones":
         if(isset($_REQUEST['selected'])) {
             foreach($_REQUEST['selected'] as $key => $data) {
+			
+			$sql = "SELECT mac_id FROM endpointman_line_list WHERE ext = " . $_REQUEST['selected'][$key];
```

Comment:
scary SQL here. This is/can be considered an SQL injection point. You need to prepare the data before you accept it from users.

Reply:
https://github.com/FreePBX-ContributedModules/endpointman/pull/64/commits/1ce6d6038b7ad648db8b401290877cc45a258a9c

Reply:
I hope this is a correct solution to prevent this :-)